explorer/info.go

```diff
@@ -0,0 +1,158 @@
+package explorer
+
+import (
+	"go.sia.tech/core/consensus"
+	"go.sia.tech/core/types"
+)
+
+// ChainStats contains a bunch of statistics about the consensus set as they
+// were at a specific block.
+type ChainStats struct {
+	Block             types.Block
+	ValidationContext consensus.ValidationContext
```

Comment:
Probably don't want to be encoding an entire ValidationContext as JSON. We can add a `/chain/context/` route for querying them.